docs/sunpy_stability.yaml

```diff
@@ -0,0 +1,60 @@
+cm:
+  status: dev
+  comments: Stuff
+coordinates:
+  status: dev
+  comments: Stuff
+data:
+  status: dev
+  comments: Stuff
+database:
+  status: dev
+  comments: Stuff
+extern:
+  status: dev
+  comments: Stuff
+image:
+  status: dev
+  comments: Stuff
+instr:
+  status: dev
+  comments: Stuff
+io:
+  status: dev
+  comments: Stuff
+lightcurve:
+  status: dev
+  comments: Stuff
+map:
+  status: dev
+  comments: Stuff
+net:
+  status: dev
+  comments: Stuff
+physics:
+  status: dev
+  comments: Stuff
+roi:
+  status: dev
+  comments: Stuff
+spectra:
+  status: dev
+  comments: Stuff
+sun:
+  status: dev
+  comments: Stuff
+tests:
+  status: dev
+  comments: Stuff
+time:
+  status: dev
+  comments: Stuff
+timeseries:
+  status: dev
+  comments: Stuff
+util:
```

Comment:
I would drop this as well, I am not sure the stability of util is either a thing or matters.